spec/features/bars/administrator_can_delete_bar_spec.rb

```diff
@@ -35,4 +36,46 @@
 
     expect(page).to_not have_content("Delete")
   end
+
+  scenario 'bar is no longer in database' do
```

Comment:
What is this test for? Be really specific about what you're testing. Currently, this expectation can just get rolled into the `administrator can delete a bar` test.